src/main/xar-resources/modules/scan.xql

```diff
@@ -40,7 +40,7 @@ declare %private function docs:load-stored($path as xs:anyURI, $store as functio
 declare %private function docs:load-external-modules($store as function(xs:string, element()) as empty-sequence()) {
     for $uri in util:mapped-modules()
```

Comment:
(Side note: It would be nice if util:mapped-modules returned xs:anyURI instead of xs:string.)